arches_for_science/templates/views/components/plugins/init-workflow.htm

```diff
@@ -1,13 +1,97 @@
-<div class="workflow-select-plugin">
-    <div class="workflow-select-card-container">
-        <!-- ko foreach: {data: workflows, as: 'workflow'} -->
-        <a data-bind="attr: {href: workflow.url, title: workflow.name}" href="#"><div data-bind="style: {'background-color': workflow.bgColor}" class="workflow-select-card">
-            <h4 class="workflow-select-title" data-bind="text: workflow.name"></h4>
-            <div data-bind="style: {'background-color': workflow.circleColor}" class="workflow-select-wf-circle">
-                <span><i data-bind="attr:{class: ('fa '+workflow.icon +' workflow-select-wf-icon')}"></i></span>
+{% load i18n %} 
+
+<div>
+    <!-- Modal -->
+    <!-- ko if: shouldShowIncompleteWorkflowsModal() -->
+    <div 
+        style="
+            position: absolute;
+            width: 100%;
+            height: 100%;
+            background: rgba(0, 0, 0, .5);
+            z-index: 1;
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            cursor: pointer;
+        "
+    >
+        <div 
```

Comment:
At about 125% zoom I start to lose information:
![Screenshot 2023-10-20 at 10 20 05 AM](https://github.com/archesproject/arches-for-science/assets/38668450/43ba7538-20b6-4ecc-a2d5-b3321a97bc57)